File: fibonacci.rb
#!/usr/bin/env ruby

require "time"

def fib(n)
  n <= 2 ? 1 : fib(n - 2) + fib(n - 1)
end

platform = RUBY_PLATFORM == "java" ? "jruby" : "ruby."

start = Time.now
fib(40)
puts "#{platform}.... #{(Time.now.to_f - start.to_f).round(6)} seconds"

File: fibonacci.py
#!/usr/bin/env python3
import time


def fib(n):
    return 1 if n <= 2 else fib(n - 2) + fib(n - 1)


start = time.perf_counter()
fib(40)
print(f"python... {round(time.perf_counter() - start, 6)} seconds")
